app/assets/stylesheets/application.css

```diff
@@ -460,102 +447,71 @@ ol.category_tags {
 	margin: 0;
 }
 
+.stories > li + li {
+	margin-top: 0.5rem;
+}
+
 ol.comments {
 	margin-bottom: 0;
 }
-ol.comments > li {
-	display: grid;
-	grid-column: begin / end;
-	grid-template-columns: subgrid;
-	row-gap: 0.25rem;
-}
-ol.comments > li + li {
-	margin-top: 0.4rem;
-}
-ol.comments1 {
-	padding-left: 0;
-}
-ol.comments.comments1 {
-	border-left-color: transparent;
-}
-ol.comments1 > li.comments_subtree > ol.comments {
-	margin-left: 0;
-	padding-left: 0;
-}
 
-
-#inside .comment ~ div > ol.comments {
-	grid-column: main / end;
-	display: grid;
-	grid-template-columns: [begin] max-content [main] 1fr [end];
-	column-gap: 0.3rem;
+ol.comments > li + li {
+	margin-top: 0.5rem;
 }
 
-div.comment_gutter {
+.comments_gutter {
+	flex-basis: var(--logo-col-width);
+	flex-shrink: 0;
 	display: flex;
 	flex-direction: column;
 	align-items: center;
-	grid-column: begin;
-	grid-row-end: span 2;
 }
 
-div.voters {
-	margin-top: 0px;
-	justify-self: center;
-	display: flex;
-	flex-direction: column;
-	align-items: center;
+.comments_body {
+	flex-grow: 1;
 }
 
 .upvoter:before {
 	content: "\25b2";
 	color: var(--color-fg-shape);
 	font-size: 12pt;
 }
-.upvoted .upvoter:before,
+.comments_gutter:has(~ .comments_body > .comment.upvoted) .upvoter:before,
```

Comment:
I saw that this was a draft but had to peek.

We can copy the upvoted class to a closer element if it helps keep selectors simple. I'd rather not lean on `:has` if we can avoid it, given your research on what's [widely available](https://github.com/lobsters/lobsters/blob/71b8e1458a9234fa2644c6e65b83ab8d4f8998ce/app/assets/stylesheets/application.css#L4).